### PR TITLE
Don't even try to run test_performance on GHA VMs

### PR DESCRIPTION
**Pull request by steven-johnson**
Even running serially, too much flakiness.

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -447,10 +448,12 @@ jobs:
         fi
 
         # tutorial has some performance measurements that can be flaky if we run them in parallel
-        TEST_GROUPS_SERIAL="performance tutorial"
+        TEST_GROUPS_SERIAL="tutorial"
```
Comment (abadams) [inline .github/workflows/test.yml]: There's one performance test in the tutorials I think. How about changing it to not be an error in this PR?

Comment (steven-johnson) [reply .github/workflows/test.yml]: also at least one in `auto_schedule` (see https://github.com/halide/Halide/issues/4749). Not sure what to do about that one.

**Comment by steven-johnson** [comment]
PTAL

File: tutorial/lesson_16_rgb_run.cpp [hunk 2]
```diff
@@ -23,6 +23,14 @@
 
 #include "halide_benchmark.h"
 
+#define check_timing(faster, slower) \
```
Comment (abadams) [inline tutorial/lesson_16_rgb_run.cpp]: The advantage of the macro is that you get the variable names in the printf, but I think we should avoid it in a tutorial. People might be thrown by the stringification of arguments and think they're not understanding something important.

There's a printf just before each assert already, so how about just:
```
void check_timing(double faster, double slower) {
  if (faster < slower) {
    fprintf(stderr, "Warning: performance was worse than expected. %f should be less than %f\n", faster, slower);
  }
}
```

Comment (steven-johnson) [reply tutorial/lesson_16_rgb_run.cpp]: Done

**Comment by steven-johnson** [comment]
PTAL

**Pull request merged**